### ISSUE TITLE
Wooden Slabs still use Pickaxes

### ISSUE THREAD

**Issue by viyic** [2zqa/AnnoyanceFix#1]
Axes still aren't effective against Wooden Slabs!

**Comment by 2zqa** [comment]
You're right! This problem might be harder to tackle since in more recent versions, minecraft just introduced a new block. I am not sure if it's easy to change tools to be more efficient based on damage values, but I will give it a try soon.
